app/scripts/controllers/eventDetails.js

```diff
@@ -485,6 +485,18 @@ angular
           validationErrorsHint = eventInformationPageHint;
         }
 
+        if (
+          $scope.conference.cruEvent &&
+          $scope.conference.ministry &&
+          $scope.getEventTypes().length !== 0 &&
+          !$scope.conference.eventType
+        ) {
+          validationErrors.push(
+            'Please enter which Event Type is applicable for this event.*',
```

Comment:
Is this `*` at the end necessary?

Reply:
<img width="735" alt="Screenshot 2023-07-28 at 11 36 23 AM" src="https://github.com/CruGlobal/conf-registration-web/assets/47907831/34bf04e4-df88-49c3-be19-9f409372b6a5">
The other error messages have them too.